src/primitives/sector/styled.rs

```diff
@@ -86,38 +110,42 @@ where
     type Item = Pixel<C>;
 
     fn next(&mut self) -> Option<Self::Item> {
+        let outer_threshold = self.outer_threshold;
+
         loop {
-            match self.state {
-                IterState::Arc => {
-                    if let Some((point, distance)) = self.iter.next() {
-                        let color = if distance < self.inner_threshold {
-                            self.inner_color
-                        } else if distance < self.outer_threshold {
-                            self.outer_color
-                        } else {
-                            None
-                        };
-
-                        if let Some(color) = color {
-                            return Some(Pixel(point, color));
+            let (point, distance) = self
+                .iter
+                .find(|(_, distance)| *distance < outer_threshold)?;
+
+            // TODO: only scale point once
```

Comment:
Should this be handled in this PR or in a future one? If it's purely for performance, should we start tracking these things with `// PERF: ...` comments, like the MSRV ones, perhaps?

Reply:
This will also be included in the next PR, so it might not be worth changing it now. But I like the idea of using PERF for longer term comments.